davidpdrsn/CardistryIO
# Add Turbolinks

I have looked through all the javascript and made some changes. Issue is that Turbolinks only evaluates the js on the initial page load, but when following links. Thus binding to `DOMContentLoaded` or `window.onload` wont work. You have either use event delegation (which they recommend) or bind to `turbolinks:load` like so

``` coffee
document.addEventListener "turbolinks:load", ->
  # ...
```

There still still something wrong with timeline.js.coffee. Doesn't work 😞

How to test:
- Navigate to a page that isn't the feed, such as a user's profile.
- Reload the page.
- Click the logo to get back to the feed.
- Scroll down. The cards should fade in as you scroll.

@bendobos wanna look into the issues with the timeline? Would be cool if we could get it working again 😊 
## Missing
- [ ] Rebase with master and update relative time replacement in javascript to not bind to dom ready.
